src/ontology/cl-edit.owl

```diff
@@ -27043,6 +27047,38 @@ EquivalentClasses(obo:CL_3000004 ObjectIntersectionOf(obo:CL_0000111 ObjectSomeV
 SubClassOf(obo:CL_3000004 obo:CL_0000101)
 SubClassOf(obo:CL_3000004 obo:CL_0000111)
 
+# Class: obo:CL_4023003 (nuclear chain fiber)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/B978-0-323-39632-5.00024-4") Annotation(oboInOwl:hasDbXref "PMID:139469") obo:IAO_0000115 obo:CL_4023003 "A type of intrafusal muscle fiber that have nuclei arranged in a linear row, and the equatorial region is not obviously expanded. are responsible for the detection of changes in muscle length. They are innervated by static gamma motor neurons and are principally associated with type II sensory fiber")
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023003 <http://orcid.org/0000-0001-7258-9596>)
```

Comment:
Suggested edits to text def:

A type of intrafusal muscle fiber that has nuclei arranged in a linear row.  Unlike nuclear bag fibers, the equatorial region of these fibers (in the centre of the spindle) is not expanded. These fibers are responsible for the detection of changes in muscle length. They are innervated by static gamma motor neurons and are principally associated with type II sensory fibers.